src/renderer/ThemeBackend.js

```diff
@@ -0,0 +1,225 @@
+var Color = require('color')
+
+function changeContrast (colorString, factor) {
+  // TODO make the black check code work
+  const color = Color(colorString).hex() === '#000000' ? Color('#010101') : Color(colorString)
+  if (color.isDark()) {
+    console.log('dark')
+    return color.lighten(factor).rgb().string()
+  } else if (color.isLight()) {
+    console.log('light')
```

Comment:
console.log, maybe a log.debug if it's not too spammy?

Reply:
the plan is to remove it after we fixed a problem with this function